config/test.yml.erb

```diff
@@ -0,0 +1,73 @@
+# Unit tests need to stub a lot of CDO properties so don't freeze config.
```

Comment:
What part of the code/config makes it not freeze config for test?

Reply:
oh this was a leftover comment (I had toyed with the idea of making `freeze` configurable, then decided to always freeze for simplicity), thanks for pointing it out, I will remove.